api/spec/controllers/spree/api/checkouts_controller_spec.rb

```diff
@@ -308,7 +308,9 @@ module Spree
       context "#{action}" do
         context "with order in confirm state" do
           subject do
-            api_put action, params
+            ActiveSupport::Deprecation.silence do
```

Comment:
Why this? Is this testing a deprecated action? May be nice to have a code comment here why we decided to silence.